js/debug/admin.js

```diff
@@ -187,3 +187,11 @@ JOBCENTRE.renderCreditCard = function(){
         creditcard(true);
     });
 };
+
+JOBCENTRE.calculateEmployer = function(){
+    $('[name=IndustryIds]').change(function () {
+        if(this.value === '33' && this.checked){
+            $("#IsRecruitAgency>option:eq(1)").prop('selected', true);
```

Comment:
Can we put a simple notice to the user that a change was made?  Maybe something like?
```
                    JOBCENTRE.alertFloater.show({
                        summary: 'Employer type changed to \'recruiter\' because you selected the \'Staffing, Recruiting, HR Outsourcing\' industry.',
                        duration: 6000
                    });
```